build/config/BUILDCONFIG.gn

```diff
@@ -245,7 +245,12 @@ if (target_os == "android") {
 } else if (target_os == "chromeos" || target_os == "linux") {
   # See comments in build/toolchain/cros/BUILD.gn about board compiles.
   if (is_clang) {
-    _default_toolchain = "//build/toolchain/linux:clang_$target_cpu"
+    if (is_cobalt) {
```

Comment:
I think we should reuse the is_cobalt block below and take advantage of string_split (on `:`) and string_join to prefix the target name with `cobalt_`

Reply:
I can add it to the block below. But the toolchain im adding in a new folder cobalt/build/toolchain/linux/BUILD.gn
as we discussed - better to not modify upstream files and add it under cobalt/